src/util.ts

```diff
@@ -994,7 +999,10 @@ export class Util {
             }
         }
         if (leftmostRange) {
-            return this.createRangeFromPositions(leftmostRange.start, rightmostRange.end);
+            //if we don't have a rightmost range, use the leftmost range for both the start and end
+            return this.createRangeFromPositions(
+                leftmostRange.start,
+                rightmostRange ? rightmostRange.end : leftmostRange.end);
```

Comment:
This is changed behavior; previously an error would have been thrown in this case. I don't know whether this case would occur in practice.